src/mqt/qecc/cococo/layouts.py

```
@@ -0,0 +1,337 @@
+        xmax += 1
+        ymin -= 1
+        ymax += 1
+
+        # collect outer ring nodes
+        corridor_nodes: list[pos] = []
+        for x in range(xmin, xmax + 1):
+            corridor_nodes.extend(((x, ymin), (x, ymax)))
+        for y in range(ymin, ymax + 1):
+            corridor_nodes.extend(((xmin, y), (xmax, y)))
+        for y in range(ymin, ymax + 1):
+            corridor_nodes.append((xmin - 1, y))  # one further col in order to be sure that corridor  # noqa: FURB113
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `FURB113`)

Remove unused `noqa` directive

(RUF100)